pacman/model/graphs/application/application_graph.py

```diff
@@ -151,50 +151,16 @@ def edges(self):
             for partition in self.outgoing_edge_partitions
             for edge in partition.edges]
 
-    def add_outgoing_edge_partition(self, edge_partition):
+    def _add_outgoing_edge_partition(self, edge_partition):
         """ Add an edge partition to the graph.
 
         Will also add any edges already in the partition as well
 
-        :param ApplicationEdgePartition edge_partition:
```

Comment:
Is there a reason for removing the parts of the docstring describing the argument? It still looks correct. The argument is still there. (The `:raises` clause can go.)

Reply:
oops put back in.